src/main/java/Deadline.java

```diff
@@ -0,0 +1,14 @@
+public class Deadline extends Task{
```

Comment:
```suggestion
public class Deadline extends Task {
```
Maybe it is better to leave a space after class name for better clarity?